src/file/file.service.spec.ts

```diff
@@ -49,44 +68,91 @@ describe("FileService", () => {
     expect(fileService).toBeDefined();
   });
 
-  it("should return success response while uploading supported file", async () => {
-    const mockResponse: UploadFileResponse = {
-      fileId: "3dd6d207-066c-42e5-bbed-f40ce7355941",
-      message: "git-cheatsheet.pdf file is uploading...",
-    };
-
-    const FileInput: FileUpload = {
-      filename: "git-cheatsheet.pdf",
-      mimetype: "application/pdf",
-      encoding: "",
-      createReadStream: jest.fn(),
-    };
-
-    jest.spyOn(fileRepository, "create").mockReturnValue({
-      id: "3dd6d207-066c-42e5-bbed-f40ce7355941",
+  describe("uploadFile mutation", () => {
+    it("should return success response while uploading supported file", async () => {
+      const mockResponse: UploadFileResponse = {
+        fileId: "3dd6d207-066c-42e5-bbed-f40ce7355941",
+        message: "git-cheatsheet.pdf file is uploading...",
+      };
+
+      const FileInput: FileUpload = {
+        filename: "git-cheatsheet.pdf",
+        mimetype: "application/pdf",
+        encoding: "",
+        createReadStream: jest.fn(),
+      };
+
+      jest.spyOn(fileRepository, "create").mockReturnValue({
+        id: "3dd6d207-066c-42e5-bbed-f40ce7355941",
```

Comment:
All mock data can be moved to mocked file

Reply:
Sure

Reply:
commits 
- [9fb2967](https://github.com/parthc2fo/upload-download-thing/pull/5/commits/9fb29679453e2741ad232427ec43855b199b24c4)
- [777e34a](https://github.com/parthc2fo/upload-download-thing/pull/5/commits/777e34ab79c8d1caafa285ded879152b5cf8cb36)